app/src/test/java/com/mertceyhan/bitcoinmarket/features/market/ui/MarketViewModelTest.kt

```diff
@@ -40,11 +39,11 @@ class MarketViewModelTest {
 
     private val marketViewStateLiveDataValues = arrayListOf<MarketViewState>()
 
-    private val layoutViewStateLiveDataObserver = mockk<Observer<LayoutViewState>>()
+    private val layoutViewStateLiveDataObserver = mockk<Observer<com.mertceyhan.bitcoinmarket.base.ui.LayoutViewState>>()
```

Comment:
Could you add it as import to the top of the file?

`import com.mertceyhan.bitcoinmarket.base.data.State`